### ISSUE TITLE
Missing xsi:type on certain observation values

### ISSUE THREAD

**Issue by bminghelli** [drajer-health/eCRNow#609]
RCKMS is seeing a relatively small, but increasing, number of errors while processing incoming eICRs due to the fact that certain `value` elements are missing the required `xsi:type` attribute.  Without the attribute, the unmarshalling of the document cannot be completed since the type cannot be derived.

Here is a sample snippet showing an `observation` within the Results section that has the issue (comments added by us):
```
<observation classCode="OBS" moodCode="EVN">
	<templateId root="2.16.840.1.113883.10.20.22.4.2"/>
	<templateId extension="2015-08-01" root="2.16.840.1.113883.10.20.22.4.2"/>
	<id extension="https://fhir-ecr.cerner.com/r4/8e95a1ff-1887-4987-a791-2170776e0067/Observation/L-4910971348/_history/1" root="2.16.840.1.113883"/>
	<code code="65222-2" codeSystem="2.16.840.1.113883.6.1" codeSystemName="LOINC">
		<originalText>
			<reference value="#labTestName7"/>
		</originalText>
	</code>
	<statusCode code="completed"/>
	<effectiveTime value="20231023141700+0000"/>
	<!--RCKMSDEV-1141
		The missing xsi:type attribute for the date value below causes RCKMS to error out-->
	<value value="20231019"/>
	<!--******** The value should be formatted similar to the following: -->
	<!--<value xsi:type="TS">20231019</value>-->
</observation>
```

See the attached (redacted) CDA document
[RCKMSDEV1141 example 1.xml.txt](https://github.com/drajer-health/eCRNow/files/13244727/RCKMSDEV1141.example.1.xml.txt)

In the sample of instances we reviewed, they all looked to be caused by the same "Date and time of symptom onset" observation (65222-2).

**Comment by mmatiasalt** [comment]
Hi @bminghelli, This issue is under review. Are you able to load screenshot examples as well as the loading query where this occurs? Thank you.

**Comment by bminghelli** [comment]
@mmatiasalt I am an RCKMS developer so I only have access to the downstream CDA document that we received and that I attached. I was hoping that someone might be able to spot a pattern with that "date and time of symptom onset" observation that seemed to be a common denominator in all of these cases and track it down from that.  Maybe the provider running ecrNow can provide what you're looking for?  Sorry, I'm not sure what else I can provide to help but if additional CDAs would be helpful, I can do that.

**Comment by mmatiasalt** [comment]
@bminghelli no worries, thank you for that information. We will continue reviewing. Thanks!

**Comment by nbashyam** [comment]
Fixed in 3.1.3

**Comment by bminghelli** [comment]
Hello, we still encounter these errors on a daily basis. I see that it was marked fixed in 3.1.3. Has that been deployed, or is it expected that we'd still them because not all clients have upgraded?
